examples/context/motivation-solution.js

```diff
@@ -0,0 +1,43 @@
+// highlight-range{1}
+const ColorContext = React.createContext();
```

Comment:
Do you think we could stick to our `ThemeContext` example?
I think `ColorContext` is a little bit less clear (why would a color need to be global? what color?)

(Note: I know it's based on old docs.)

Reply:
Looking again, I see an example below uses Theme. It would be great to have a consistent example throughout this doc instead of using two very similar but different examples. Can you unify them?